packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -91,26 +95,19 @@ export class CXLMarketingNavElement extends LitElement {
       <nav>
         <slot></slot>
       </nav>
-
-      <vaadin-device-detector
-        @wide-changed="${(e) => {
-          /**
-           * Initial page load doesn't seem to trigger `wide` attribute reflection.
-           *
-           * @see https://github.com/Polymer/lit-element/issues/549
-           * @see https://polymer.slack.com/archives/C03PF4L4L/p1581109849195000
-           * @see https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/queueMicrotask
-           */
-          const { wide } = e.target;
-
-          Promise.resolve().then(() => {
-            this.wide = wide;
-          });
-        }}"
-      ></vaadin-device-detector>
     `;
   }
 
+  constructor() {
+    super();
+
+    this.addController(
+      new MediaQueryController(this._wideMediaQuery, (matches) => {
+        this.wide = matches;
+      })
```

Comment:
Are we also looking at some sort of a CXLWideMediaQueryMixin idea here?

Reply:
> Are we also looking at some sort of a CXLWideMediaQueryMixin idea here?
> 
> Can be a new PR.

Yes, can be `CXLWideMediaQueryMixin` in new PR.

Reply:
@lkraav This would be a very easy update. Could we do it in a new PR?